### PR TITLE
[KYUUBI #1610] Rest frontend service jetty server has started at null

### LINKED ISSUE #1610: [Bug] Rest frontend service jetty server has started at null.
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the bug

we shall print the actual bind-address 

### Affects Version(s)

master

### Kyuubi Server Log Output

```logtalk
Started ServerConnector@31b33e91{HTTP/1.1, (http/1.1)}{10.242.189.214:10099}
2021-12-23 11:59:56.620 INFO server.Server: jetty-9.4.41.v20210516; built: 2021-05-16T23:56:28.993Z; git: 98607f93c7833e7dc59489b13f3cb0a114fb9f4c; jvm 1.8.0_251-b08
2021-12-23 11:59:57.393 INFO handler.ContextHandler: Started o.e.j.s.ServletContextHandler@3bd3d05e{/,null,AVAILABLE}
2021-12-23 11:59:57.393 INFO server.Server: Started @3342ms
2021-12-23 11:59:57.393 INFO server.KyuubiRestFrontendService: Rest frontend service jetty server has started at null.
```

### Kyuubi Engine Log Output

_No response_

### Kyuubi Server Configurations

```yaml
kyuubi.frontend.protocols=THRIFT_BINARY,REST
```

### Kyuubi Engine Configurations

_No response_

### Additional context

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by yaooqinn**
### _Why are the changes needed?_

fix #1610
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

> 2022-01-21 15:10:53.999 INFO server.KyuubiRestFrontendService: KyuubiRestFronten
dService has started at 10.242.189.214:10099

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/JettyServer.scala [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    connector: ServerConnector,
+    rootHandler: ContextHandlerCollection) {
+  def start(): Unit = {
+    server.start()
+    connector.start()
+    server.addConnector(connector)
+  }
+
+  def stop(): Unit = {
+    server.stop()
```
Comment (SteNicholas) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/JettyServer.scala:39]: Should the server stop after the connector stops?

Comment (yaooqinn) [reply kyuubi-server/src/main/scala/org/apache/kyuubi/server/JettyServer.scala:39]: make sense, thanks

**Review by SteNicholas — APPROVED** [review]
LGTM.

**Comment by yaooqinn** [comment]
thanks, merged to master

**Pull request closed without merging**